gridstatus/ercot.py

```diff
@@ -453,24 +473,37 @@ def _get_todays_outlook_non_forecast(self, date, verbose=False):
 
         return data.reset_index(drop=True)
 
-    def get_load_forecast(self, date, verbose=False):
-        """Returns load forecast
+    def get_load_forecast(self, date, forecast_type: SevenDayLoadForecast, verbose=False):
+        """Returns load forecast.
+
+        Currently only supports today's forecast, at the system-level.
 
-        Currently only supports today's forecast
+        Arguments:
+            date (str): time to download. Returns last hourly report
+                before this time. Supports "latest".
+            forecast_type (SevenDayLoadForecast): The load forecast type to return.
+                Enum of possible values. 
+            verbose (bool, optional): print verbose output. Defaults to False. 
         """
+
+
         if not utils.is_today(date, self.default_timezone):
             raise NotSupported()
 
         # intrahour https://www.ercot.com/mp/data-products/data-product-details?id=NP3-562-CD
         # there are a few days of historical date for the forecast
         today = pd.Timestamp.now(tz=self.default_timezone).normalize()
-        doc_info = self._get_document(
-            report_type_id=SEVEN_DAY_LOAD_FORECAST_BY_FORECAST_ZONE_RTID,
+        
+        doc = self._get_document(
+            report_type_id=forecast_type.value,
             date=today,
```

Comment:
since we're revisiting this function, let's model it after the `get_hourly_wind_report` and use `published_before=None if date == "latest" else date` rather than date. 

when we do this, we also need to remove this line above that rounds the supplied date to the beginning of the day

```python
today = pd.Timestamp.now(tz=self.default_timezone).normalize()
```